### PR TITLE
2213 calendar

### PR DESCRIPTION
**Pull request by emitche**
Adds basic calendar to site

### CODE REVIEW

File: app/assets/javascripts/admin/calendar/dependencies.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+//= require moment
+//= require fullcalendar
```
Comment (linesarefuzzy) [inline app/assets/javascripts/admin/calendar/dependencies.js]: Why have these dependencies in a separate file? You can put them at the top of `calendar.js.coffee`—just change the syntax to `#=` for coffeescript.

File: app/assets/javascripts/admin/calendar/calendar.js.coffee [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$ ->
+  $('#calendar').fullCalendar()
```
Comment (linesarefuzzy) [inline app/assets/javascripts/admin/calendar/calendar.js.coffee]: Yay coffeescript!

**Comment by linesarefuzzy** [comment]
Merge away after minor change suggested above. Thanks!

File: app/assets/javascripts/admin/calendar/dependencies.js (hunk 1 above)
Comment (emitche) [inline app/assets/javascripts/admin/calendar/dependencies.js]: That makes sense. Will make that change.

**Pull request merged**